lib/archethic/transaction_chain/transaction/validation_stamp/ledger_operations.ex

```diff
@@ -506,10 +569,12 @@ defmodule Archethic.TransactionChain.Transaction.ValidationStamp.LedgerOperation
     {unspent_outputs, rest} =
       reduce_unspent_outputs(rest, nb_unspent_outputs, [], protocol_version)
 
-    {nb_consumed_inputs, rest} = rest |> VarInt.get_value()
-
-    {consumed_inputs, rest} =
-      reduce_unspent_outputs(rest, nb_consumed_inputs, [], protocol_version)
+      {consumed_inputs, rest} = if protocol_version < 5 do
```

Comment:
In node release 1.4.5 we upgraded protocol version to 5. In this PR you should update protocol version to 6, and this control would be based on version < 6

Reply:
Done